File: terceiroSemestre/poo/polimorfismo/python/ex2/contato.py
class Contato:
    def __init__(self, nome, endereço, email):
        self.nome = nome
        self.endereço = endereço
        self.email = email
        
    def __str__(self):
        print("Nome :" + self.nome)
        print("Endereço: " + self.endereço)
        print("email: " + self.email)

        

File: 3rdPeriod/ObjectOrientedProgramming/polimorfismo/java/src/Contato.java
public class Contato {
    private String nome;
    private String endereço;
    private String email;
    
    Contato(String nome, String endereço, String email){
        this.nome = nome;
        this.endereço = endereço;
        this.email = email;
    }

    String toString(){
        System.out.println("Nome: " + nome);
        System.out.println("Endereço: " + endereço);
        System.out.println("email: " + email);
    }
    
}
